packages/mpv.rb

```diff
@@ -4,21 +4,23 @@ class Mpv < Package
   description 'Video player based on MPlayer/mplayer2'
   homepage 'https://mpv.io/'
   @_ver = '0.34.1'
-  version @_ver
+  version "#{@_ver}-1"
   license 'LGPL-2.1+, GPL-2+, BSD, ISC and GPL-3+'
-  compatibility 'x86_64 armv7l aarch64'
+  compatibility 'armv7l aarch64 x86_64'
   source_url 'https://github.com/mpv-player/mpv.git'
   git_hashtag "v#{@_ver}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mpv/0.34.1_armv7l/mpv-0.34.1-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mpv/0.34.1_armv7l/mpv-0.34.1-chromeos-armv7l.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mpv/0.34.1_x86_64/mpv-0.34.1-chromeos-x86_64.tar.zst'
+    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mpv/0.34.0_i686/mpv-0.34.0-chromeos-i686.tpxz',
```

Comment:
Version doesn't match?

Reply:
Accidental build failure. I'll start a rebuild for i686...

Reply:
Fixed.